components/Learn/PreviousNextLink/__tests__/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import PrevNextLink from '..';
+import messages from '../../../../i18n/locales/en.json';
```

Comment:
It would be best if you did not import the messages. Usage of IntlProvider should just be:

```<IntlProvider onError={() => {}}>```

Sometimes you don't even need to import the IntlProvider, eg, if you're only using `useLocale` you can mock it.

Reply:
Component not rendering during test execution without importing messages as the properties referenced are not being found and getting "MISSING_TRANSLATION" exception